lib/ansible/cli/galaxy.py

```diff
@@ -305,9 +305,9 @@ def execute_info(self):
                 role_info.update(role_spec)
 
             data = self._display_role_info(role_info)
-            ### FIXME: This is broken in both 1.9 and 2.0 as
-            # _display_role_info() always returns something
-            if not data:
+            
+            role_location = os.path.join(roles_path[0], role)
```

Comment:
roles_path is not from roles, it is an Ansible configuration that defines the directories in which Ansible will look for roles (aside from roles/ adjacent to play). Not to be confused with `role_path` which returns the path to the current role.